src/routes/tools/_common/stores.js

```diff
@@ -91,7 +91,6 @@ export const locationStore = (() => {
     subscribe,
     updateLocation: (location, isUpload = false) =>
       update((store) => {
-        if (!location) return;
```

Comment:
Note to self: keep an eye out to see if this causes any bugs in other tools.

Reply:
Also note to self: this caused a bug in the `locationStore` as any of the update methods should return the store, which wasn't happening here.